src/App.test.tsx

```diff
@@ -30,6 +30,9 @@ const aUser = {
 	},
 };
 
+// Extend the timeout, as we are getting failed test because of timeout
+jest.setTimeout(30000);
+
```

Comment:
30 seconds seems excessive and suggests there is another issue here, perhaps unrelated to Emotion?

Reply:
Its causing CI tests to fail and its kinda outside the scope (likely and API issue causing the slowness?) . I will drop it to a lower number somewhere between 3-5seconds

Reply:
I think if this api call is suddenly taking a long time and it only happens on this branch then we need to understand why so that we don't risk passing through a real issue into production